src/shared-components/button/components/roundButton/__snapshots__/test.js.snap

```diff
@@ -196,8 +196,8 @@ exports[`<RoundButton /> UI snapshots renders with props 1`] = `
   right: 0;
   top: 0;
   margin-top: -6px;
-  width: 38px;
   opacity: 0;
+  width: 38px;
   width: 36px;
```

Comment:
It appears this doubly-applied width already existed, so this diff is just because I alphabetized props, presumably.